packages/script/src/errors.ts

```diff
@@ -16,7 +11,8 @@ const printLineWithId = (id: string, line: string) =>
   `${id === ZeroBytes32 ? 'script' : id}: ${line}`;
 
 export class ScriptResultDecoderError extends Error {
-  constructor(result: TransactionResult<'failure'>, message: string) {
+  logs: any[];
```

Comment:
Could this be anything other than `any`?

Also, should we reconsider our lint configs in this regard?